src/Tizen.NUI/src/public/BaseComponents/ViewBindableProperty.cs

```diff
@@ -607,6 +607,40 @@ public partial class View
             return null;
         });
 
+        /// <summary>
+        /// ClockwiseFocusableViewProperty
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty ClockwiseFocusableViewProperty = BindableProperty.Create(nameof(View.ClockwiseFocusableView), typeof(View), typeof(View), null, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var view = (View)bindable;
+            if (newValue != null) { view.ClockwiseFocusableViewId = (int)(newValue as View)?.GetId(); }
```

Comment:
Maybe minor but... don't we need to set `-1` if newValue is fail to convert as `View` class?